src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -3325,6 +3337,20 @@ private int encodeSectionValue(
             return progress;
         }
 
+        private int encodeSectionValueNull(
+            int deferred,
+            DirectBuffer buffer,
+            int offset,
+            int limit)
+        {
+            AmqpDescribedTypeFW descriptor =
+                amqpDescribedTypeRW.wrap(valueBuffer, 0, valueBuffer.capacity()).set(VALUE).build();
```

Comment:
Suggest pre-creating and reusing `VALUE` descriptor instead of making extra copies.